crates/agentgateway/proto/resource.proto

```diff
@@ -538,6 +538,43 @@ message PolicySpec {
     FailureMode failure_mode = 2;
   }
 
+  // Logging configuration policy
+  message Logging {
+    // CEL expression to filter out logs. If unset, all logs are considered.
+    google.protobuf.StringValue filter = 1;
+
+    // Additional log fields configuration
+    message Fields {
+      // Field keys to remove from logs
+      repeated string remove = 1;
+      // Field keys to add with CEL expressions
+      map<string, string> add = 2;
+    }
+
+    // Additional metric fields configuration
+    message MetricFields {
+      // Field keys to add with CEL expressions
+      map<string, string> add = 1;
+    }
+
+    // Fields applied to logs
+    Fields fields = 2;
+    // Fields applied to metrics
+    MetricFields metric_fields = 3;
+    // Metric names to exclude entirely
+    repeated string excluded_metrics = 4;
+
+    // Comma separated levels or per-target level string. Empty uses default info level
+    string level = 5;
```

Comment:
Level is a global singleton. We cannot have it in a policy. I think we probably should not have it declarative at all for now

(same with format)